sonar-csharp-plugin/src/main/resources/documentation.md

```diff
@@ -33,15 +33,9 @@ As an alternative, an `.editorconfig` file can be used to disable the analysis f
 dotnet_diagnostic.Sxxx.severity = none
 ```
 
-## Improving Performance  
-
-You can enable parallel processing to speed up analysis and reduce overall build time by setting the following environment variable:
-```
-SONAR_DOTNET_ENABLE_CONCURRENT_EXECUTION=true
-```
-If you still have problems [this](https://community.sonarsource.com/t/47279) community thread may help you diagnose the issues.
-
 ## Related Pages
+* [Investigating the performance of .NET Analysis](https://community.sonarsource.com/t/the-sonarsource-guide-for-investigating-the-performance-of-net-analysis/47279)
```

Comment:
The short-hand version can be used
```suggestion
* [Investigating the performance of .NET Analysis](https://community.sonarsource.com/t/47279)
```

Reply:
Sorry I didn't see your suggestion in time. Also, I didn't know there was a shorthand. TIL!